### PR TITLE
PR: added typography component

### PR DESCRIPTION
**Pull request by abhisekgupta001**
Hi, added some text utilities, please review and leave some suggestions.

- headings types
- paragraph types
- gray texts
- centered texts

### headings types
image

### paragraph types
image

### gray texts
image

### centered texts
image

### CODE REVIEW

File: public/components/text_utilities/text_util.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            </li>
+          </ul>
+        </div>
+      </aside>
+      <main class="doc__main">
+        <section class="image-section doc-section" id="image-section">
+          <div class="section__heading--container">
+            <h1 class="section__title">Typography</h1>
+            <p class="section__para">
+              There are multiple varaiation of a text on web page.
```
Comment (muskaanshah) [inline public/components/text_utilities/text_util.html]: There's a typo in this line

**Review by muskaanshah — APPROVED** [review]
Looks good, just that one typo

File: public/components/text_utilities/text_util.html (hunk 1 above)
Comment (abhisekgupta001) [reply public/components/text_utilities/text_util.html]: Thanks. will fix this

**Review by Tarandeep-s1ngh — APPROVED** [review]
Looks good!

**Pull request merged**